packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -402,7 +402,7 @@ export function TransferPanelMain({
       isTokenArbitrumOneNativeUSDC(selectedToken.address) ||
       isTokenArbitrumSepoliaNativeUSDC(selectedToken.address)
     ) {
-      updateUSDCBalances(selectedToken.address)
+      updateUSDCBalances()
```

Comment:
I don't think we need to await here because it's not relevant to the following logic?